pkg/gui/view_helpers.go

```diff
@@ -173,9 +173,7 @@ func (gui *Gui) switchFocus(g *gocui.Gui, oldView, newView *gocui.View) error {
 }
 
 func (gui *Gui) resetOrigin(v *gocui.View) error {
-	if err := v.SetCursor(0, 0); err != nil {
-		return err
-	}
+	_ = v.SetCursor(0, 0)
```

Comment:
we are we now ignoring this error?

Reply:
Yes I've spent 3 hours trying to figure out why lg crashes every time the boxes have 0 lines of usable space. And after lots of attempts to get this right this seems to be the solution.  Because this is the only error

Reply:
Fair enough, I just hope it doesn't make it harder for us to debug things down the line. Could we instead make the gocui methods more lenient in our fork?

It also seems that we still get a panic if we set the height of the window to e.g. 0, which I believe is not an issue on the master branch.

Reply:
When i run:
```sh
cd vendor/github.com/jesseduffield
mv gocui gocui-bak
git clone https://github.com/mjarkk/gocui && cd gocui
git checkout fix-to-small-lazygit-panic
```
After doing that there seem to be no panics, see: https://github.com/jesseduffield/gocui/pull/6 for more info.